R/data-module.R

```diff
@@ -217,6 +217,13 @@ setMethod("groups", "DataModule", function(object) {
     .groups(modulePath(object))
 })
 
+#' @exportS3Method tidyfr::groups
```

Comment:
please remove the `tidyfr::` here.

Reply:
It's kind of "required" by the `@exportS3Method`, see https://cran.r-project.org/web/packages/roxygen2/vignettes/namespace.html

Reply:
Just an update, sticking to the S3 is possible, as well as adding the S4 generic, but we have to stick using `object` argument as input of the `groups` function, otherwise there will be conflict/issues with the manual, since the `groups` function depends on the `DataModule` class manual. So if you prefer to be consistent with the S3 syntax we should probably move the `groups` method man entry somewhere else, otherwise we can stick to the `object` argument in the input.

Reply:
I'd suggest to use `x` instead of `object` if we use the S3 method. `object` is generally only used for `S4` methods. I don't think we have to move the man page somewhere else. I would still keep it in the `DataModule` and simply add a documentation entry for `@param x`. For the user it doesn't matter is he/she uses a S3 or S4 method, but having all documentation in one place is easier for them.